src/caf/distribute/gravity_model/multi_area.py

```diff
@@ -563,29 +676,35 @@ def _gravity_function(
         return achieved_residuals
 
     # pylint:enable=too-many-locals
-    def run(self):
+    def run(
+        self,
+        distributions: list[MultiCostDistribution],
+        running_log_path: Path,
+        furness_tol=DEFAULT_FURNESS_TOL,
```

Comment:
Missing type annotation
```suggestion
        furness_tol: float = DEFAULT_FURNESS_TOL,
```